app.rb

```diff
@@ -95,15 +95,15 @@ class InvalidChannelName < StandardError; end
     redirect full_url
   end
 
-  get /(?<channel>\/[\w]+)?\/(?<project>[\w-]+)\/metadata/ do
+  get /(?<channel>\/[\w]+)?\/(?<project>[\w-]+)\/metadata.?(?<format>\w+)?/ do
```

Comment:
Let's not introduce a new feature here. The format extension stuff should work for __all__ endpoints when we introduce it.